nix/nixos/cardano-node-service.nix

```diff
@@ -25,11 +25,6 @@ let
             "--signing-key ${cfg.signingKey}"}"
           "${lib.optionalString (cfg.delegationCertificate != null)
             "--delegation-certificate ${cfg.delegationCertificate}"}"
-          "--tracing-verbosity-${cfg.tracingVerbosity}"
```

Comment:
the `tracingVerbosity` option definition could be removed too.

Reply:
@jbgi removed. This should pass ci now. Once it does are we good to merge: https://github.com/input-output-hk/iohk-nix/pull/303